flowforge/materials/__init__.py

```diff
@@ -0,0 +1,8 @@
+from .Fluid import Helium
+
+material_database = {
+    # SOLIDS
+
+    # FLUIDS
+    "helium": Helium
```

Comment:
will this conflict?

Reply:
Yes, but after merging #25 I will take care of any merge conflicts.

Reply:
ready to update now

Reply:
Done.